### PR TITLE
feat(#1886): add README for custom_goals integration test

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
Add `README.md` for `custom_goals` integration test.

Closes: #1886

### CODE REVIEW

**Comment by volodya-lombrozo** [comment]
@mximp Could you have a look, please?

File: eo-maven-plugin/src/it/custom_goals/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<img alt="logo" src="https://www.objectionary.com/cactus.svg" height="100px" />
+
+The purpose of `custom-goals` integration test is to check partial compilation
+of EO programs. We have to be sure that the plugin correctly performs particular
+set of goals in order to compile a program. We just check that for a
+particular set of goals the particular set of artifacts and folders will be
+compiled and create. The idea to use only specific goals of the plugin rather
```
Comment (mximp) [inline eo-maven-plugin/src/it/custom_goals/README.md:7]: @volodya-lombrozo "...and created...",   "...the idea is.."

Comment (volodya-lombrozo) [reply eo-maven-plugin/src/it/custom_goals/README.md]: @mximp Thanks. Fixed.

**Review by mximp — APPROVED** [review]

**Comment by yegor256** [comment]
@volodya-lombrozo thanks!

**Pull request merged**